content/news/2025-11-06-fosdem2026-sdr-dsp-call/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "FOSDEM 2026 SDR/DSP Devroom – Call for Participation"
+author: "SDR/DSP Devroom Organizers"
+date: "2025-11-06"
+categories: ["conference", "fosdem", "sdr", "dsp"]
+aliases: ["/news/fosdem2026-sdr-dsp-call"]
+banner: conference_web.svg
+---
+
+# Call for Participation: SDR/DSP Devroom at FOSDEM 2026
+
+The **Software Defined Radio / Digital Signal Processing (SDR/DSP)** devroom is being organized for **FOSDEM 2026**, held in **Brussels, Belgium**, on **Sunday, February 1, 2026**.
+
+We are seeking contributions discussing **free and open-source software (FOSS)** for RF discrete-time **digital signal processing** and **software-defined radio (SDR)**. Topics of interest include:
```

Comment:
Should we also include other things than RF DSP (hearing aids, seismography, DSP for quantum computers)?